cluster-configuration/services-configuration.yaml

```diff
@@ -87,7 +87,7 @@ restserver:
   # port for rest api server
   server-port: 9186
   # secret for signing authentication tokens, e.g., "Hello PAI!"
-  jwt-secret: your_jwt_secret
+  jwt-secret: "Hello PAI!"
```

Comment:
Will jet-secret be Hello PAI!, or “Hello PAI!” (literal string with double-quotes)?

Reply:
@YundongYe@outlook.com what is the correct format?

---
In reply to: [195367368](https://github.com/Microsoft/pai/pull/727#discussion_r195367368) [](ancestors = 195367368)

Reply:
Updated

---
In reply to: [195699006](https://github.com/Microsoft/pai/pull/727#discussion_r195699006) [](ancestors = 195699006,195367368)